sources/ascwp/src/config.rs

```diff
@@ -0,0 +1,60 @@
+use crate::error::{self, Result};
+use serde::Deserialize;
+use snafu::ResultExt;
+use std::fs;
+use std::path::{Path, PathBuf};
+const DEFAULT_CONFIG_PATH: &str = "/testConfig.toml";
```

Comment:
It looks like we set a separate default below. Should we remove this one? Or centralize the default to this file?